doc/api/configuration.md

```diff
@@ -240,4 +240,23 @@ print(res)
 %config SqlMagic.style = "SINGLE_BORDER"
 res = %sql SELECT * FROM languages LIMIT 2
 print(res)
-```
\ No newline at end of file
+```
+
+## `persist_snippets`
+
+Default: `False`
+
+Stores the snippets as SQL files
+
+```{code-cell} ipython3
```

Comment:
I think we can add `{versionadded}` here

Reply:
Added this